### PR TITLE
Feature/70

### PR DESCRIPTION
**Pull request by TienDucHo**
In this PR I have:
- Created a Header Component that contains the logo of ISA
- Created a Vendor Card component that takes in vendor's information as props and return a vendor card with styling according to the design on Figma
- Added Mock Data Helper for the Vendor List
- Created a Home View with the Header and a Flat List that renders the Vendor's data into Vendor Card
- Added snapshot test for both Home View and Vendor Card Component

Please review my code @dimassudjito

### CODE REVIEW

File: mobile/App.js [hunk 1]
```diff
@@ -8,12 +8,12 @@
...
 import React from 'react';
 import {SafeAreaProvider} from 'react-native-safe-area-context';
-import ScanView from './src/views/ScanView';
+import HomeView from './src/views/HomeView';
 
 const App = () => {
   return (
     <SafeAreaProvider>
-      <ScanView></ScanView>
+      <HomeView></HomeView>
```
Comment (dimassudjito) [inline mobile/App.js:16]: I'm gonna assume you'll bring back ScanView in the ticket where you do the bottom navbar

File: mobile/src/components/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
+    justifyContent: 'center',
+    alignItems: 'center',
+    width: '100%',
+    height: 80,
```
Comment (dimassudjito) [inline mobile/src/components/Header.jsx:20]: I think we can trim down the header by a little bit (e.g., 70-75), its a bit too thick now

**Comment by dimassudjito** [comment]
**Reviewer's checklist**

General Check

- [ ]  Check that the code compiles and runs [Dimas]
- [ ]  Check that the code is producing the required output by comparing it to the acceptance criteria in the GitHub issues. (e.g., compare it with Figma sketch for frontend work)
- [ ]  Check that the code has acceptable tests and that the tests are passing.
- [ ]  Check that the PR is directed at the correct branch (develop Vs. main)
- [ ]  Check that the PR does not have merge conflict [Dimas]
- [ ]  Check that CI/CD tests are passing

Implementation Review

- [ ]  Understand implementation approach and give feedback on how we can improve the general approach
- [ ]  Understand data structures being used in the implementation and provide feedback (i.e., is it optimal?)
- [ ]  Understand external libraries used in the implementation and provide feedback (i.e., is it deprecated? Is there better libraries out there?)
- [ ]  Check for repeated code and give feedback
- [ ]  Check for performance-intensive code (e.g., heavy algorithm) and give feedback
- [ ]  Check for dead code or commented code that should be deleted
- [ ]  Check if variable names are sensible and descriptive

**Review by dimassudjito — CHANGES_REQUESTED** [review]
Home page is looking super!

- Let's organize components into logical folder (e.g., per page + shared folder)
- move vendorMockData.js into src/data folder
- we can also delete helper/example.jsx and components/Example.jsx if it's no longer needed

File: mobile/App.js (hunk 1 above)
Comment (TienDucHo) [reply mobile/App.js:16]: It is what I am going to do during the navigation ticket

File: mobile/src/views/HomeView.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        keyExtractor={item => item.id}
+        style={styles.vendorList}
+      />
+    </View>
+  );
+};
+const styles = StyleSheet.create({
+  welcome: {
+    color: 'violet',
+  },
```
[the remark this answers is not in the document]
Comment (TienDucHo) [reply mobile/src/views/HomeView.jsx]: Deleted  with aec264e70ef3b5bbb34263489cb2ee90c2bebe5e

File: mobile/src/components/Header.jsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    borderBottomWidth: 1,
+    borderColor: '#2E6933',
```
[the remark this answers is not in the document]
Comment (TienDucHo) [reply mobile/src/components/Header.jsx]: React Navigation has Theming capabilities, I will do that during the implementation of the navbar

File: mobile/src/components/Header.jsx (hunk 2 above)
Comment (TienDucHo) [reply mobile/src/components/Header.jsx]: > Home page is looking super!
> 
> * Let's organize components into logical folder (e.g., per page + shared folder)
> * move vendorMockData.js into src/data folder
> * we can also delete helper/example.jsx and components/Example.jsx if it's no longer needed

Fixed with 07334f2bd1ea6b52d8eb9d25f5c276e298e96c0a, e1fce8ae3eb719d2653acf1688bb6cd8c756eecd and 597e25032d28eb8cd57ea428ddd169a20f9f2c9b

**Review by dimassudjito — APPROVED** [review]

**Pull request merged**